package.json

```diff
@@ -35,6 +35,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.53",
     "@researchgate/react-intersection-observer": "^1.0.0",
+    "canvas": "^2.6.1",
```

Comment:
Can you say more about this dependency that is added?

Reply:
Sorry was testing something, just removed it